hummingbird/ml/_parse.py

```diff
@@ -48,10 +52,34 @@ def parse_sklearn_api_model(model):
     # One global scope is enough for parsing scikit-learn models.
     scope = topology.declare_scope("__root__")
 
-    # Declare input variables. Sklearn always gets as input a single dataframe,
-    # therefore by default we start with a single `input` variable
+    # Declare input variables.
     inputs = []
-    inputs.append(scope.declare_local_variable("input"))
+    n_inputs = extra_config[constants.N_INPUTS] if constants.N_INPUTS in extra_config else 1
+    if constants.TEST_INPUT in extra_config:
+        from onnxconverter_common.data_types import FloatTensorType, DoubleTensorType, Int32TensorType, Int64TensorType
+
+        test_input = extra_config[constants.TEST_INPUT] if n_inputs > 1 else [extra_config[constants.TEST_INPUT]]
+        for i in range(n_inputs):
+            input = test_input[i]
+            if input.dtype == np.float32:
+                input_type = FloatTensorType(input.shape)
+            elif input.dtype == np.float64:
+                input_type = DoubleTensorType(input.shape)
+            elif input.dtype == np.int32:
+                input_type = Int32TensorType(input.shape)
+            elif input.dtype == np.int64:
+                input_type = Int64TensorType(input.shape)
```

Comment:
Is `str` the only thing we don't support here?

Reply:
yea